vcl_templates/www.vcl.erb

```diff
@@ -248,6 +248,17 @@ sub vcl_recv {
     set req.http.original-url = req.url;
   }
 
+  if (req.url.path ~ "(?i)^/register-to-vote-armed-forces") {
+    set req.url.path = "/register-to-vote-armed-forces";
```

Comment:
`req.url.(basename|dirname|ext|path|qs)` are all read-only, computed values based on `req.path` (see e.g. documentation for [`req.url.path`](https://docs.fastly.com/vcl/variables/req-url-path/). `set req.url = ...` is the only line required, which will both set the canonical path, and remove any querystring.